src/main/HomeScreen.js

```diff
@@ -43,12 +44,14 @@ export default function HomeScreen(props: Props): Node {
     <SafeAreaView mode="padding" edges={['top']} style={styles.wrapper}>
       <OfflineNoticePlaceholder />
       <View style={styles.iconList}>
-        <TopTabButton
+        <TopTabButtonGeneral
           name="globe"
           onPress={() => {
             dispatch(doNarrow(HOME_NARROW));
           }}
-        />
+        >
+          <Icon size={24} style={{ textAlign: 'center ' }} color={BRAND_COLOR} name="globe" />
```

Comment:
Good to split this NFC change out as its own commit -- that's a helpful simplification for the main commit.

For that to do its job of making the changes easier to read and understand, though, it's important that each individual commit is correct and coherent:
https://zulip.readthedocs.io/en/latest/contributing/version-control.html

As it stands, the app actually crashes at this commit. If you run Flow while at this commit, it'll point out the cause of that crash, and also another type error.